Cargo.toml

```diff
@@ -10,6 +10,7 @@ topos-crypto = { path = "./crates/topos-crypto", default-features = false }
 
 # Various utility crates
 clap = { version = "4.0", features = ["derive", "env", "string"] }
+lazy_static = "1"
```

Comment:
Should we upgrade our Rust version and use this instead?
https://doc.rust-lang.org/nightly/std/sync/struct.LazyLock.html

For reference:
https://github.com/rust-lang-nursery/lazy-static.rs/issues/214

Reply:
As it is an unstable feature (nightly only) we can't use that for now. [ref](https://github.com/rust-lang/rust/issues/109736)